ui-src/src/actions/index.js

```diff
@@ -1,116 +1,148 @@
 // Holochain actions
-export const GET_HANDLE = 'getHandle'
-export const GET_HANDLES = 'getHandles'
-export const GET_FOLLOW = 'getFollow'
-export const APP_PROPERTY = 'appProperty'
-export const POST = 'post'
-export const GET_POST = 'getPost'
-export const POST_MOD = 'postMod'
-export const FOLLOW = 'follow'
-export const GET_POSTS_BY = 'getPostsBy'
-export const GET_AGENT = 'getAgent'
-export const NEW_HANDLE = 'newHandle'
-export const UNFOLLOW = 'unfollow'
+export const GET_HANDLE = "getHandle";
+export const GET_HANDLES = "getHandles";
+export const GET_FOLLOW = "getFollow";
+export const APP_PROPERTY = "appProperty";
+export const POST = "post";
+export const GET_POST = "getPost";
+export const POST_MOD = "postMod";
+export const FOLLOW = "follow";
+export const GET_POSTS_BY = "getPostsBy";
+export const GET_AGENT = "getAgent";
+export const NEW_HANDLE = "newHandle";
+export const UNFOLLOW = "unfollow";
+export const TOGGLE_MODAL = "toggleModal";
```

Comment:
TOGGLE_MODAL could be separated out from the "Holochain actions", since it's a UI only action, and has no backend implications. Just by putting it above the "Holochain actions" comment above, and giving it a "UI actions" comment or something

Reply:
done